.github/workflows/unit-tests.yml

```diff
@@ -158,6 +158,9 @@ jobs:
           COVERAGE: ${{ env.COVERAGE == 'true' && needs.retrieve-selected-tests.outputs.tests == ''}}
         run: bash -l build_tools/azure/test_script.sh
 
+      - name: Run doc tests
+        run: bash -l build_tools/azure/test_docs.sh
```

Comment:
We could do the same as Azure and skip this when `[all random seeds]` marker is used, but let's keep it simple for now ...

We can revisit later if needed.